src/api/projects/queries/getProject/index.js

```diff
@@ -19,12 +22,49 @@ const resolver = async (
     throw new Error(AUTHENTICATION_ERROR_NO_USER)
   }
 
-  const project = await Project.query()
+  const project: Project = await Project.query()
     .eager(
-      '[locales, translationKeys.translationValues, translationKeys.translationValues.locale]'
+      '[translationKeys.translationValues, translationKeys.translationValues.locale]'
     )
     .findOne({ slug: projectSlug, ownerId: request.user.id })
 
+  project.locales = await Locale.query()
```

Comment:
This is used to make sure we load locales in the right order (with default one on top of the list)